partials/protocols.html

```diff
@@ -0,0 +1,81 @@
+{% macro displayProtocol(protocol, protocolVersion) %}
+    {% if protocolVersion %}
+    <span class="bg-teal-500 font-bold no-underline text-white uppercase rounded ml-2"
+          style="height: 20px;font-size: 11px;padding: 3px;">{{ protocol }}
+        {{ protocolVersion }}</span>
+    {% else %}
+    <span class="bg-teal-500 font-sans font-bold no-underline text-white uppercase rounded ml-2"
+          style="height: 20px;font-size: 11px;padding: 3px;">{{ protocol }}</span>
+    {% endif %}
+{% endmacro %}
+
+
+{% macro bindings(propName, prop, odd=false) %}
+
+    {% if prop.hasBindings() %}
+        {% for protocol in prop.bindingProtocols() %}
+        <div class="{% if open %}is-open{% endif %} mb-4">
```

Comment:
usage of `mt-4` make it look much more aligned, have a look

![Screenshot 2021-04-12 at 11 00 10](https://user-images.githubusercontent.com/6995927/114369139-6190db80-9b7e-11eb-92ea-118ac3eb1963.png)

Reply:
Done ; i do uniformization to align all elements (even in the collapsed blocks)

I think that we need to develop a macro for those blocks (in order to keep the same style over all rendering blocks)